modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+= TinyMCE 6.2
+:navtitle: TinyMCE 6.2
+:description: Release notes for TinyMCE 6.2
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.2 was released for {enterpriseversion} and {cloudname} on Monday, September 26^th^, 2022. It includes {productname} 6.2, additional changes to Premium plugins, and several new Premium plugins. These release notes provide an overview of the changes for {productname} 6.2:
+
+* xref:new-premium-plugins[New Premium plugins]
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+* xref:deprecated[Deprecated]
+* xref:known-issues[Known issues]
+
+[[new-premium-plugins]]
+== New Premium plugins
+
+=== Spelling Autocorrect 1.0.0
```

Comment:
Should this be at the bottom of the list so its alphabetical?

Reply:
To quote my commit message:

> Moved Spelling Autocorrect to end of New Premium plugins section. The three new premium plugins are now presented in alphabetical order.